pkg/domain/infra/abi/images.go

```diff
@@ -264,3 +264,23 @@ func (ir *ImageEngine) Pull(ctx context.Context, rawImage string, options entiti
 // 	copy(report.Report.Id, id)
 // 	return &report, nil
 // }
+func (ir *ImageEngine) Load(ctx context.Context, opts entities.ImageLoadOptions) (*entities.ImageLoadReport, error) {
+	var (
+		writer io.Writer
+	)
+	if !opts.Quiet {
+		writer = os.Stderr
```

Comment:
Writing to os.Stderr from this layer? abi shouldn't be doing any presentation work.

Reply:
The writing happens in c/image for the progress bars during copy.

Reply:
@vrothberg Okay, I'll write a future issue on that.